src/Padawan/Framework/Domain/Project/Persister.php

```diff
@@ -26,9 +26,10 @@ public function __construct(PathResolver $path, LoopInterface $loop)
     public function save(Project $project)
     {
         $this->checkForPadawanDir($project->getRootFolder());
-        return $this->fs->file(
-            $this->getProjectIndexFilePath($project->getRootFolder())
-        )->putContents($this->serialize($project));
+        return file_put_contents(
```

Comment:
oh, I see, looks like problems with reactphp/filesystem adapters. Well, I'm afraid of braking interface, before `file_put_contents` this method used to return `PromiseInterface`.